lib/src/spinner/Spinner.tsx

```diff
@@ -109,13 +73,14 @@ const DxcSpinner = ({
   );
 };
 
-const determinatedValue = (value: SpinnerPropsType["value"], strokeDashArray:number) => {
+const determinedValue = (value: SpinnerPropsType["value"], strokeDashArray: number) => {
```

Comment:
I think the correct word determinate. Determine means to "figure out".

Reply:
But, determinated is an adjective: "to be determinated". A certain value is a determined value.

https://www.linguee.es/ingles-espanol/traduccion/determined.html
https://www.linguee.es/ingles-espanol/traduccion/to+be+determinated.html

Reply:
I checked other design sytems and it seems that the word used is determinate.

https://carbondesignsystem.com/components/progress-bar/usage/
https://carbondesignsystem.com/components/progress-bar/usage/

Reply:
Updated.